http/http.go

```diff
@@ -32,18 +31,44 @@ type stats struct {
 }
 
 func init() {
+	http.HandleFunc("/debug/pprofstats", Stats)
+}
+
+// router
+type router interface {
```

Comment:
Which other third party packages can satisfy this interface out of the box? My suggestion was to avoid adding more abstractions but to provide an http/mux package with a mux specific implementation.
```
package mux

func AttachProfiler(r *mux.Router) {}
```

There is an issue filed for martini (see https://github.com/rakyll/gom/issues/17) and I guess other frameworks can follow the same strategy to provide attacher utilities.